### PR TITLE
CODE-1579

### PR DESCRIPTION
**Pull request by aimurphy**
repo mirroring & adjust text to reflect that repos can be at 3 scopes.

[Onboarding guide](https://65f1ebd50579a20d8ecb27c5--harness-developer.netlify.app/docs/code-repository/get-started/onboarding-guide)
[Mirror repositories](https://65f1ebd50579a20d8ecb27c5--harness-developer.netlify.app/docs/code-repository/config-repos/mirror)

Example of text change for multiple scopes:

image

### CODE REVIEW

File: docs/code-repository/config-repos/mirror.md [hunk 1]
```diff
@@ -0,0 +1,307 @@
+---
+title: Mirror repositories
+description: Create pipelines to sync branch and tag changes across repos.
+sidebar_position: 30
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Mirroring syncs changes from one repo to another. In Harness, you can create CI pipelines that mirror branch and tag changes.
```
Comment (johannesHarness) [inline docs/code-repository/config-repos/mirror.md:10]: `In Harness, you can create CI pipelines that mirror branch and tag changes from one repo to another.` sounds better i'd say

File: docs/code-repository/config-repos/mirror.md [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+
+You can set up one-way or two-way mirroring.
+
+## Configure mirroring in Harness
+
+In Harness, mirroring uses a mirror script in a CI pipeline. This script replicates commits from the source repo in the target repo.
```
Comment (johannesHarness) [inline docs/code-repository/config-repos/mirror.md:16]: Is it `from the source repo on the target repo` - `in the target repo` sounds off? (might also just be my english)

Comment (aimurphy) [reply docs/code-repository/config-repos/mirror.md:16]: I think this is one of those phrasings that depends on how you conceptualize repos (boxes vs lists). I think either could be correct?

File: docs/code-repository/config-repos/mirror.md [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+```
+
+</TabItem>
+</Tabs>
+
+### Add the mirror script
+
+This script mirrors changes from one source repo to one target repo. Later, you'll create input sets and triggers that populate the variables in the script with the values for your source and target repos.
+
+To use this pipeline to mirror changes from one source to multiple targets (or multiple pairs of sources and targets), you can create [input sets](#create-input-sets) and [triggers](#create-triggers) for each source-target combination.
```
Comment (johannesHarness) [inline docs/code-repository/config-repos/mirror.md:111]: should this part be in the mirror script section, too? we already have similar things on the initial pipeline section and the input variables section.

File: docs/code-repository/config-repos/mirror.md [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+
+1. In your mirror pipeline, select **Triggers** in the Pipeline Studio header.
+2. Select **New Trigger** and select the **Custom** webhook trigger.
+
+   While Harness offers provider-specific webhook triggers, to ensure the pipeline works for any combination of SCM providers, you must use [custom webhook triggers](/docs/platform/triggers/trigger-deployments-using-custom-triggers).
+
+3. Enter a trigger **Name** and select **Continue**. The **Description** and **Tags** are optional.
+4. Skip the **Conditions** tab and go to the **Pipeline Input** tab.
+5. Select the [input set](#create-input-sets) containing the values for `sourceToken`, `targetToken`, `sourceURL`, and `targetURL` for the repos you'll mirror with this trigger.
+6. Set `syncDelete` to either `true` or `false` depending on the desired behavior.
```
Comment (johannesHarness) [inline docs/code-repository/config-repos/mirror.md:253]: syncDelete is the last input field, should we put swap it with below list element?

File: docs/code-repository/config-repos/mirror.md (hunk 3 above)
Comment (aimurphy) [reply docs/code-repository/config-repos/mirror.md:111]: Cleaned up as a result of making input sets optional.

**Pull request merged**